ballista-examples/Cargo.toml

```diff
@@ -29,7 +29,7 @@ publish = false
 rust-version = "1.56"
 
 [dependencies]
-arrow-flight = { version = "6.0.0" }
+arrow-flight = { version = "6.1.0" }
```

Comment:
I think these versions need to be something like `^6.1.0` to force using arrow 6.1.0 or newer

Reply:
@Dandandan mentioned that in my previous PR that by default cargo treats `6.1.0` as `^6.1.0`: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
maybe we should use "6.1" instead of "6.1.0" to make it less restrictive.

Reply:
yeah, that's a good idea.

Reply:
FTR, They are all functionally equivalent.
`=` and `^` both mean semver compatible. Adding the final zero has no effect (but might be more clear to remove it)

Source:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
ha, you are right. patch version only makes sense if we need it to be higher than a particular patch release, i.e. greater than 0.